src/libraries/Microsoft.PowerFx.Core/Public/CodeActionRange.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerFx.Core.Public
+{
+    /// <summary>
+    /// Code action fix range location.
+    /// </summary>
+    public class CodeActionRange
```

Comment:
>CodeActionRange [](http://example.com/codeflow?start=17&length=15)

Can we use our existing Span object?

Reply:
Refactored to move new quickfix interface under protocol project.